### ISSUE TITLE
Output file

### ISSUE THREAD

**Issue by anekha** [GaussianObject/GaussianObject#54]
Hello 
Wondering if anyone has any insight into the 3D output file generated.
I believe the rendered file is saved in 'output/gs_init/kitchen/test/ours_None and output/gs_init/kitchen/render/ours_None'
I have pulled out the .ply file but it just looks like a point cloud with no rendering. Am I getting something wrong - I wanted to get the 3D file and work with it.
![Screenshot 2024-10-03 at 10 53 23](https://github.com/user-attachments/assets/eceb1dce-c16f-441b-b3ef-8ac81774f909)

**Comment by chensjtu** [comment]
Sorry for the late response! In fact, the ply in the ours_None is a 3DGS file, which can not be directly viewed by Meshlab. For better visualization, you may need a web viewer or a local viewer. You may use [https://niujinshuchong.github.io/mip-splatting-demo/](https://niujinshuchong.github.io/mip-splatting-demo/) or [https://github.com/yzslab/gaussian-splatting-lightning](https://github.com/yzslab/gaussian-splatting-lightning) to render this 3DGS file.

**Comment by anekha** [comment]
Thank you. Do you have any idea if we can refine the output and change it into a .obj or other 3D file that can be used in other applications?

**Comment by chensjtu** [comment]
Yes, how to produce mesh from 3DGS is a hot research topic. We have tried SUGAR to extract mesh, but the performance is terrible. Maybe using 2DGS as the 3D representation or using the coarse 3DGS to form a mesh and then binding one 3DGS on each face of the coarse mesh, then optimizing this hybrid 3D representation can achieve fine-granted mesh.

**Comment by 7rwang** [comment]
Hi,
do we need `mkdir output/gs_init/kitchen ` manually?

**Comment by chensjtu** [comment]
Of course not; code will generate this folder automatically~

**Comment by 7rwang** [comment]
> Of course not; code will generate this folder automatically~

Thanks a lot for ur reply! Could you check this issue?[https://github.com/chensjtu/GaussianObject/issues/62](url)

**Comment by chensjtu** [comment]
> > Of course not; code will generate this folder automatically~
> 
> Thanks a lot for ur reply! Could you check this issue?[https://github.com/chensjtu/GaussianObject/issues/62](url)

yes, I'll check this latter. Sorry for the late reply!

**Comment by 7rwang** [comment]
> > > Of course not; code will generate this folder automatically~
> > 
> > 
> > Thanks a lot for ur reply! Could you check this issue?[https://github.com/chensjtu/GaussianObject/issues/62](url)
> 
> yes, I'll check this latter. Sorry for the late reply!

No problem, thanks for ur help in advance!!
